Library/Homebrew/test/support/fixtures/cask/Casks/with-pkg-target-current-user-home-directory.rb

```diff
@@ -0,0 +1,11 @@
+cask "with-pkg-target-current-user-home-directory" do
+  version "1.2.3"
+  sha256 "8c62a2b791cf5f0da6066a0a4b6e85f62949cd60975da062df44adf887f4370b"
+
+  url "file://#{TEST_FIXTURE_DIR}/cask/MyFancyPkg.zip"
+  homepage "https://brew.sh/fancy-pkg"
+
+  pkg "MyFancyPkg/Fancy.pkg", target: "CurrentUserHomeDirectory"
+
+  uninstall pkgutil: "my.fancy.package"
```

Comment:
I copied both these fixtures from the `with-installable` fixture, but I simplified the `uninstall` stanza, since all the extra stuff there isn't relevant to the specs these are used in.